unicore/comments/service/models.py

```diff
@@ -16,11 +16,14 @@
     (u'removed_by_moderator', u'Removed by moderator'),
     (u'removed_by_community', u'Removed by community'),
     (u'removed_for_profanity', u'Removed for profanity'))
+COMMENT_STREAM_STATES = (u'open', u'closed', u'disabled')
```

Comment:
Tiny preference for making these upper case since it's mostly intended to be treated as a constant?